tests/requirements-test.txt

```diff
@@ -0,0 +1,2 @@
+pytest==5.4.3
```

Comment:
Why not including those requirements in the original requirements file?

Reply:
i agree with  shay , it has no any relation to the application.